.github/workflows/push.yml

```diff
@@ -31,12 +31,15 @@ env:
     connection.keboola.com|8781|${{ secrets.TEST_KBC_PROJECT_8781_TOKEN }};
     connection.keboola.com|8782|${{ secrets.TEST_KBC_PROJECT_8782_TOKEN }};
 
-  # S3 repository
-  IAM_USER: keboola-as-code-dist-s3-upload
-  AWS_BUCKET_NAME: keboola-as-code-dist
-  AWS_REGION: us-east-1
-  AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
-  AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+  # Legacy S3 repository
+  LEGACY_IAM_USER: keboola-as-code-dist-s3-upload
+  LEGACY_AWS_BUCKET_NAME: keboola-as-code-dist
+  LEGACY_AWS_REGION: us-east-1
+  LEGACY_AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+  LEGACY_AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+  # S3 Repository in KAC Assets Account
+  CLI_DIST_AWS_BUCKET_NAME: cli-dist-keboola-com
```

Comment:
@michaljurecko Chtěl bych aby ses podíval zejména tady na ten poslední commit (https://github.com/keboola/keboola-as-code/pull/872/commits/d2b948e4eaabe0b22acf0cb201769142212ded71).

AWS credentials legacy S3 jsem přejmenoval (oprefixoval `LEGACY_`) a předávají se do steps který slouží k legacy uploadu manuálně. Kromě toho se nově volá `aws-actions/configure-aws-credentials@v1` akce, která zařídí assume role přes OIDC a vyexportuje AWS env vars (`AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY` a `AWS_SESSION_TOKEN`).

Reply:
ok, presiel som to a nevidim tam problem